package.json

```diff
@@ -992,6 +992,20 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-users.profilepictures.all",
+        "displayName": "Users: Can view, edit, and delete profile pictures",
+        "description": "Also includes basic permissions to edit user profile",
+        "subPermissions": [
+          "ui-users.profilepictures.view",
```

Comment:
Use `profile-picture` in place of `profilepicture`. Petty, sorry; developers are so cranky about names 😅 Be consistent with the name already established in the back-end permissions. Alas, consistency on single vs plural is a battle I lost years ago. Curse DHH and his awful Rails ORM naming conventions, the worst decision he ever made.